third_party/benchmark_requirements.txt

```diff
@@ -0,0 +1,43 @@
+six==1.16.0
+matplotlib==3.9.2
+mpld3==0.5.10
+alabaster==0.7.12
+Babel==2.14.0
+bandit==1.6.2
+beautifulsoup4==4.11.1
+breathe==4.33.1
+bs4==0.0.1
+certifi==2024.07.04
+chardet==3.0.4
+clang-format==15.0.7
+colorama==0.4.1
+docutils==0.15.2
+exhale==0.3.0
+idna==3.7
+imagesize==1.1.0
+Jinja2==3.1.4
+lxml==4.9.3
+Mako==1.3.0
+MarkupSafe==2.1.5
+packaging==24.1
```

Comment:
FYI, the issue we had with `six` installation, was because `packaging` was too old (<20.0)